var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -141,20 +145,40 @@ def configure(self):
 
     @when('@4')
     def configure(self):
-        configure('-fast',
-                  '-no-webkit',
-                  *self.common_config_args)
+        options = self.common_config_args[:]
+
+        if "darwin" in sys.platform:
+            options.extend(
+                ['-fast',
+                 # XXX(osx): sdk stuff; should depend on the architecture and the SDKs available.
+                 '-arch', 'x86_64',
+                 '-sdk', '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk',
```

Comment:
it's is safer to get the path by
```
$ xcrun --show-sdk-path
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk
```
So no need to hard-code it. You can add function to utilities somewhere in spack just in case it will be needed by other packages.